core/micro-services/workflow-core/src/main/resources/workflow-core-config.yaml

```diff
@@ -0,0 +1,10 @@
+storage:
+  result-storage-mode: mongodb
+  mongodb:
+    url: "mongodb://localhost:27017"
+    database: "texera_storage"
+    commit-batch-size: 1000
+  jdbc:
+    url: "jdbc:mysql://0.0.0.0:3306/texera_db?serverTimezone=UTC"
```

Comment:
use localhost (the loopback address) by default

Reply:
changed